src/i18n/en.ts

```diff
@@ -549,6 +549,16 @@ const en = {
         "public.dashboard.actions.login": "Log in",
         "public.dashboard.actions.register": "Create account",
         "public.dashboard.actions.vocabularies": "Browse vocabularies and terms",
+
+        "comments.title": "Comments",
+        "comments.create.placeholder": "Type your comment...",
+        "comments.create.submit.title": "Submit",
+        "comments.list.empty": "No comments, yet.",
+        "comments.comment.like": "Like this comment",
+        "comments.comment.like.on": "Cancel reaction",
```

Comment:
Misaligned with the czech version:
"comments.comment.like.on": "Už se mi nelíbí",
"comments.comment.like.on": "Cancel reaction"

Maybe change the Czech version to "Zrušit reakci", until we come up with some consistent naming for the actions (with meanings - like / cancel like / dislike / cancel dislike - but better, more understandable)